tests/cli/test_utils.py

```diff
@@ -197,10 +204,24 @@ def test_get_prompt_analyzer():
     assert analyzer.variables == {"foo": "bar"}
 
 
+@pytest.mark.parametrize(
```

Comment:
what about if the version tag is first? or in the middle?

Reply:
no difference, cuz i find index and replace existing as long as its first one

Reply:
added anyways